src/views/masq_activating_modal.dot

```diff
@@ -0,0 +1,40 @@
+<div class="modal__masq_activating">
+  <div class="modal__masq_activating__header">
+    <div class="icon-masq-by-qwant modal__masq_activating__logo"></div>
+    <i class="icon-x modal__masq_activating__close" {{= click(this.close, this) }}></i>
+  </div>
+
+  <div class="modal__masq_activating__body">
+    {{? this.activating}}
+      <div class="modal__masq_activating__animation">
+        <video autoplay loop src="{{= window.location.origin }}/statics/images/masq/gifs/masq_loading.mp4"></video>
```

Comment:
Using "origin" is incorrect here: the "baseUrl" ( /maps/...) is missing.  
Actually an absolute path can directly be used in templates, since a `<base>` tag is defined in the html head: 
```suggestion
        <video autoplay loop src="/statics/images/masq/gifs/masq_loading.mp4"></video>
```

Reply:
:+1: